mat64/symmetric_test.go

```diff
@@ -119,6 +119,15 @@ func (s *S) TestSymAdd(c *check.C) {
 			}
 		}
 	}
+
+	method := func(receiver, a, b Matrix) {
+		rd := receiver.(*SymDense)
```

Comment:
Add `type symAdder interface { AddSym(a, b Symmetric) }` then s/.(*SymDense)/.(symAdder)/.

This is superficial, but goes to indicating that this is a generic case.